genesis/src/main.rs

```diff
@@ -772,6 +798,7 @@ fn main() -> Result<(), Box<dyn error::Error>> {
     add_validator_accounts(
         &mut genesis_config,
         &mut bootstrap_validator_pubkeys.iter(),
```

Comment:
kind of strange how `add_validator_accounts` is setup. Think it would be nicer to parse out these Pubkeys into a struct rather than relying on sequential ordering in groups of 3.

definitely not a problem for this PR, just pointing it out.

Reply:
Yeah, agree this set up is a bit weird.